### PR TITLE
adding 3scale/apisonator support to power

### PR DESCRIPTION
**Pull request by shubham-bhagwat0**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: 3/3scale-apisonator/redhat_ubi8/build_info.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "maintainer": "Shubham.Bhagwat@ibm.com",
+    "package_name" : "apisonator",
+    "github_url": "https://github.com/3scale/apisonator.git",
+    "version": "2.14-stable",
+    "package_dir": "3/3scale-apisonator/redhat_ubi8",
+    "default_branch": "main",
+    "build_script": "3s_apisonator_ubi_8_7.sh",
```
Comment (nikhil-kalbande) [inline 3/3scale-apisonator/redhat_ubi8/build_info.json:8]: add build script name like `3scale-apisonator" and package name also like same

File: 3/3scale-apisonator/redhat_ubi8/build_info.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+    "maintainer": "Shubham.Bhagwat@ibm.com",
+    "package_name" : "apisonator",
```
Comment (nikhil-kalbande) [inline 3/3scale-apisonator/redhat_ubi8/build_info.json:3]: make package name like 3scale-apisonator

File: 3/3scale-apisonator/redhat_ubi8/build_info.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "github_url": "https://github.com/3scale/apisonator.git",
+    "version": "2.14-stable",
+    "package_dir": "3/3scale-apisonator/redhat_ubi8",
```
Comment (nikhil-kalbande) [inline 3/3scale-apisonator/redhat_ubi8/build_info.json:6]: package_dir should point to "3/3scale-apisonator"

File: 3/3scale-apisonator/redhat_ubi8/3s_apisonator_ubi_8_7.sh [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+if [ -d "$PACKAGE_NAME" ] ; then
+  rm -rf $PACKAGE_NAME
+  echo "$PACKAGE_NAME   | $OS_NAME | GitHub | Removed existing package if any"
+fi
+
+if ! git clone $PACKAGE_URL -b $PACKAGE_VERSION ; then
+    echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+	echo "$PACKAGE_NAME  |  $PACKAGE_URL | $OS_NAME | GitHub  | Pass |  Both_Install_and_Test_Success"
+    exit 0
```
Comment (nikhil-kalbande) [inline 3/3scale-apisonator/redhat_ubi8/3s_apisonator_ubi_8_7.sh:38]: exit 0 means success.
make it exit 1

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

**Review by shubham-bhagwat0 — COMMENTED** [review]
changes made

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**